pkg/version/version.go

```diff
@@ -32,6 +34,11 @@ type Info struct {
 	BuildDate  string `json:"buildDate"`
 }
 
+type InfoWithEtcd struct {
```

Comment:
I don't think we want to do this.  How about we build a separate version command for `openshift`.  It really is special.  It has multiple versions to show.